src/main/java/seedu/clinic/model/attribute/Email.java

```diff
@@ -43,8 +43,11 @@ public Email(String email) {
     /**
      * Returns if a given string is a valid email.
      */
-    public static boolean isValidEmail(String test) {
-        return test.matches(VALIDATION_REGEX);
+    public static boolean isValidEmail(String email) {
+        if (email.isEmpty()) {
```

Comment:
Yup, because it is now an optional field and to make UI a lot less painful I've decided to set optional fields to "".

Unless we want to remove missing fields from UI which can be quite a headache.